test/e2e_node/taints.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package scheduling
+package e2enode
```

Comment:
`test/e2e_node` uses a totally different e2e mechanism. We should move it to `test/e2e/node`.